src/name_bazaar/ui/pages/manage_names_page.cljs

```diff
@@ -77,7 +81,7 @@
                                                         (str name constants/registrar-root))])
             full-name (when (seq name)
                         (str name constants/registrar-root))
-
+            name-record @(subscribe [:public-resolver/record (namehash full-name)])
```

Comment:
In this component `(namehash full-name)` is calculated 5 times. Optimise to just 1